### ISSUE TITLE
Henry's Fork Snake - Public access

### ISSUE THREAD

**Issue by ghost** [troutspotr/troutspotr-frontend#157]
Good access to the Henry's Fork near Harriman and Island Park not shown. If there is a good reason we should document it and close this. 

<img width="372" alt="screen shot 2018-09-25 at 8 00 58 pm" src="https://user-images.githubusercontent.com/43528087/46051256-fa18fa80-c0fd-11e8-9ebb-d220dde0c9e9.png">

**Comment by andest01** [comment]
![image](https://user-images.githubusercontent.com/1424223/46108048-b3c9a700-c1a2-11e8-8af9-8efed9bfe86c.png)

@zacharyrpope confirm this is the guy.

[IDPR Managed Land.zip](https://github.com/troutspotr/troutspotr-frontend/files/2421519/IDPR.Managed.Land.zip)


Source: 
https://gis2.idaho.gov/arcgis/rest/services/DPR/IDPR_Managed_Resources/MapServer
![image](https://user-images.githubusercontent.com/1424223/46108212-17ec6b00-c1a3-11e8-9248-2aa25e1ab94e.png)

**Comment by ghost** [comment]
Yep, that looks like it. Are you saying that State Park is listed as private in the IDPR database?

**Comment by andest01** [comment]
Nah, I just didn't supplement Idaho with any unsafe state data from PADUS.
